gem/styles/gem-springster/modules/_m-tag-cloud.scss

```diff
@@ -0,0 +1,80 @@
+.tag-cloud {
+  display: inline-block;
+  width: 100%;
+  margin-top: 20px;
+
+  &__container {
+    @include divider-combo-1;
+    background-color: $sunny-yellow;
+    padding: 60px 1em 0 1em;
+    margin-top: 40px;
+  }
+
+  &__block {
+    width: 100%;
+    text-align: center;
+    font-size: 13px;
+    margin: 20px 0 0;
+    padding: 0 0 20px;
+    list-style-type: none;
+  }
+
+  &__icon {
+    background: url('/static/img/tag-cloud-love-hands.svg') no-repeat 0 0/contain;
+    padding: 2px 5px;
+    height: 88px;
+    width: 100px;
+    position: absolute;
+    left: 50%;
+    margin-left: -50px;
+  }
+
+  &__heading {
+    @include zig-zag-underline;
+    text-align: center;
+
+    h4 {
```

Comment:
I saw the tag cloud had different variations on articles as well. On clean up we must be able to pick and choose pattern elements without loosing styles.